examples/pytorch/AxHyperOptimizationPTL/conda.yaml

```diff
@@ -0,0 +1,18 @@
+channels:
+- defaults
+- conda-forge
+- pytorch
+dependencies:
+- python=3.8.2
+- pytorch>=1.6.0
```

Comment:
@ankan94 Using the conda.yaml, it complains about the the cuda version, cuda=11.0 with torchvision as it has CUDA Version=10.2

Reply:
Used the cudatoolkit=10.2 version for both. I think torchvision comes with latest cuda version 10.2.